fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -915,6 +916,11 @@ class BrowserRobot {
     fun fillPdfForm(name: String) {
         // Set PDF form text for the text box
         itemWithResId("pdfjs_internal_id_10R").setText(name)
+        // Setting the text will launch the keyboard
+        mDevice.waitForWindowUpdate(packageName, waitingTime)
+        verifyKeyboardVisibility(true)
```

Comment:
you can try the Espresso.closeSoftKeyboard(), or add an if condition, to avoid pressing back unnecessarily.

Reply:
Added a condition, but closeSoftKeyboard() proved to be unreliable, so I'm still using the device pressBack interaction